### PR TITLE
Ale branch1

### PR DESCRIPTION
**Pull request by alejandro-loja**
updated User and Product, added faker, updated update user and product seed data

### CODE REVIEW

**Review by Diane-Sayos — COMMENTED** [review]
should we do unique on username and email?

File: db/User.js [hunk 1]
```diff
@@ -1,106 +1,125 @@
...
+    validate: {
+      isEmail: true,
+    },
+  },
+  imageUrl: {
+    type: STRING,
+    // validate:{
+    //   isUrl: true,
+    // }
+  },
```
Comment (bob-zs) [inline db/User.js:32]: Maybe, see if you can make a custom` isUrl` validation with regex. 
https://github.com/sequelize/sequelize/issues/9086

File: db/User.js [hunk 2]
```diff
@@ -1,106 +1,125 @@
...
-User.prototype.addToCart = async function({ product, quantity}){
+User.prototype.addToCart = async function ({ product, quantity }) {
   const cart = await this.getCart();
   let lineItem = await conn.models.lineItem.findOne({
     where: {
       productId: product.id,
-      orderId: cart.id
-    }
+      orderId: cart.id,
+    },
```
Comment (bob-zs) [inline db/User.js:54]: 👍 by the way, thanks for the last element comma style.
I was trying to enable this with prettier. Will get this auto-prettier formatted in one of my earliest PRs as possible

File: server.js [hunk 3]
```diff
@@ -1,19 +1,39 @@
...
+    const foo = await Product.create({
+      name: "foo",
+      description: faker.lorem.paragraph(1),
+      price: faker.finance.amount(0.05, 20, 2),
+    });
+    const bar = await Product.create({
+      name: "bar",
+      description: faker.lorem.paragraph(1),
+      price: faker.finance.amount(0.05, 20, 2),
+    });
```
Comment (bob-zs) [inline server.js:31]: 👍

**Review by bob-zs — COMMENTED** [review]
Thanks so much @alejandro-loja for getting this on. This is great seeing this db and faker data seeding being set up!

I'm trying to see if I can pull this brunch and run it as well before I approve of it.

**Review by Banychen — APPROVED** [review]

File: db/User.js (hunk 1 above)
Comment (bob-zs) [reply db/User.js:32]: Delete this commented code

**Pull request merged**